doc/rst/build.rst

```diff
@@ -24,33 +24,33 @@ mpiFileUtils release packages are available as attachments from their respective
 https://github.com/hpc/mpifileutils/releases
 
 mpiFileUtils optionally depends on libarchive, version 3.5.1.
-If new enough, the system install of libarchive may be sufficient,
+If new enough or newer, the system install of libarchive may be sufficient,
```

Comment:
(nit) you don't need "or newer" here.  If you want "or newer", best to put it on line 26 after "version 3.5.1".